azurerm/resource_arm_virtual_machine.go

```diff
@@ -96,21 +96,27 @@ func resourceArmVirtualMachine() *schema.Resource {
 				MaxItems: 1,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
+						"image_id": {
```

Comment:
🤔 whilst I'm not opposed to this - would it make more sense to call this `id` to match the API Documentation / ARM Template?